cluster-http/src/main/resources/reference.conf

```diff
@@ -11,7 +11,11 @@ akka.management {
   http.route-providers += "akka.management.cluster.ClusterHttpManagement"
 
   # registers cluster healthcheck to be included in akka-management's http endpoint
-  http.route-providers += "akka.management.cluster.ClusterHealthCheck"
+  http {
```

Comment:
this should probably move out of the http config object as health checks aren't specific to http

Reply:
I agree